config/locales/de.yml

```diff
@@ -1,4 +1,14 @@
 de:
+  application_name: 100eyes
+  components:
+    chat_form:
+      placeholder: Gib hier deine Nachfrage ein..
```

Comment:
```suggestion
      placeholder: Gib hier deine Nachfrage ein…
```

Reply:
Cool, didn't know there is an extra character for ellipsis. Is it good practice to use it? Like more semantic and still available in all fonts?

Reply:
I’m pretty sure that if a font doesn’t include a separate ellipsis glyph, the character will be [decomposed](https://unicode.org/reports/tr15/pdtr15.html) into three dots. That’s some first-class unicode magic… 😉

In general, all high-quality/professionally-made fonts will include that glyph – at least I’ve never come across one that didn’t.